dataprofiler/tests/profilers/test_int_column_profile.py

```diff
@@ -230,8 +230,8 @@ def test_profiled_histogram(self):
         # with equal bin size, each bin has the width of 1
         data3 = ["1", "1", "3", "4"]  # 3 bins, range of 3
         expected_histogram3 = {
-            'bin_counts': np.array([2, 0, 2]),
-            'bin_edges': np.array([1.0, 2.0, 3.0, 4.0]),
+            'bin_counts': np.array([1, 1, 0, 2]),
+            'bin_edges': np.array([1.0, 1.75, 2.5, 3.25, 4.0]),
```

Comment:
why are we changing this test?

Reply:
The new technique suggests that this has better loss than the previous. We can discuss further.

Reply:
i'm not sure it looks better in this case, the second bin edge [1.7, 2.5] should not contain a number